src/openforms/appointments/contrib/qmatic/client.py

```diff
@@ -37,3 +37,7 @@ def request(self, method: str, url: str, *args, **kwargs):
             )
 
         return response
+
+
+def get_client():
```

Comment:
there's not much point in trying to make vendor plugins similar since they can wildly vary and the goal of the plugins is to encapsulate everything vendor-specific inside the plugin. IMO it adds unnecessary constraints.

So yes, please do just initialize each client in the plugin methods - it is effectively the same but removes one layer of indirection.

Reply:
done